netbout-web/src/test/java/com/netbout/rest/TkAppITCase.java

```diff
@@ -57,6 +58,18 @@
         Manifests.read("Netbout-Documentation");
 
     /**
+     * Xpath to the 'page not found' notice.
+     */
+    private static final String MISSING =
```

Comment:
@original-brownbear we definitely prefer inlining here, see http://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html